lib/common/asar.js

```diff
@@ -292,10 +292,9 @@
       process.nextTick(() => fs.lstat(pathArgument, callback))
     }
 
-    const { statSyncNoException } = fs
     fs.statSyncNoException = pathArgument => {
       const { isAsar, asarPath, filePath } = splitPath(pathArgument)
-      if (!isAsar) return statSyncNoException(pathArgument)
+      if (!isAsar) return fs.statSyncNoException(pathArgument)
```

Comment:
@MarshallOfSound I'm actually not quite sure what this code is _supposed_ to do. This would is just recursion, right?